### PR TITLE
Add support for .NET 8

### PR DESCRIPTION
**Pull request by toini**

### CODE REVIEW

**Review by ronanbarrett — COMMENTED** [review]
Good opportunity to start using renovate? The config you need is [here](https://www.notion.so/tibber/Software-Vulnerability-Management-Process-4f1c851d25b946048da9a82cc62865b1?pvs=4#02c8d24977dc46549342b71d1d63c7db). We also need to opt into renovatebot which I can help with if you choose to do this.

File: 8-alpine-tools/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+# Defaults New Relic
+ENV CORECLR_NEWRELIC_HOME	/usr/local/newrelic-dotnet-agent
+ENV CORECLR_PROFILER	{36032161-FFC0-4B61-B559-F6C5D41BAE5A}
+ENV CORECLR_PROFILER_PATH	/usr/local/newrelic-dotnet-agent/libNewRelicProfiler.so
+ENV NEW_RELIC_APPLICATION_LOGGING_ENABLED	false
+ENV NEW_RELIC_APPLICATION_LOGGING_FORWARDING_ENABLED	false
+ENV NEW_RELIC_DISTRIBUTED_TRACING_ENABLED	true
+
+# This variable enables New Relic (disabled by default)
```
Comment (ronanbarrett) [inline 8-alpine-tools/Dockerfile:22]: I don't understand the comment here.

File: 8-alpine/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+# Defaults New Relic
+ENV CORECLR_NEWRELIC_HOME	/usr/local/newrelic-dotnet-agent
+ENV CORECLR_PROFILER	{36032161-FFC0-4B61-B559-F6C5D41BAE5A}
+ENV CORECLR_PROFILER_PATH	/usr/local/newrelic-dotnet-agent/libNewRelicProfiler.so
+ENV NEW_RELIC_APPLICATION_LOGGING_ENABLED	false
+ENV NEW_RELIC_APPLICATION_LOGGING_FORWARDING_ENABLED	false
+ENV NEW_RELIC_DISTRIBUTED_TRACING_ENABLED	true
+
+# This variable enables New Relic (disabled by default)
```
Comment (ronanbarrett) [inline 8-alpine/Dockerfile:11]: I don't understand the comment here.

File: 8-alpine-tools/Dockerfile (hunk 1 above)
Comment (toini) [reply 8-alpine-tools/Dockerfile:22]: If CORECLR_ENABLE_PROFILING is set to 0 then new relic does not work.

**Comment by toini** [comment]
@ronanbarrett you could add a PR to replace this? https://github.com/tibber/Tibber.ExampleServiceDotnet/blob/main/.github/.dependabot.yml

File: 8-alpine-tools/Dockerfile (hunk 1 above)
Comment (ronanbarrett) [reply 8-alpine-tools/Dockerfile:22]: I see but that comment doesn't really say that. I would have said:

`# New Relic requires coreclr profiling to work (disabled by default)`

**Comment by ronanbarrett** [comment]
@toini This is the repo that actually references the base images so this is the really important one to have renovate on I guess?

**Comment by toini** [comment]
@ronanbarrett yeah if we will pin the sha in this repo. Then you can add a PR here, too?

**Pull request merged**